Frontend/src/CustomScripts/convertData.mjs

```diff
@@ -88,7 +91,50 @@ const json = { categories, defaultHistoryLength };
 
 // write this JSON object
 const output = JSON.stringify(json, null, 2);
+
+const cats = json.categories
+let newCategories = []
+
+for (let i = 0; i < cats.length; i++) {
+  let res = cats[i].category.split(';')
+
+  let c = res[0]
+  let subc = res[1]
+  let ind = undefined;
+  for (let j = 0; j < newCategories.length; j++) {
+    if (newCategories[j].category.localeCompare(c) == 0) {
+      ind = j
+      break
+    }
+  }
+
+  if (ind) {
+    newCategories[ind].subcategories.push({
+      'subcategory': subc,
+      'values': cats[i]['values']
+    })
+  } else {
+    newCategories.push({
+      'category': c,
+      'subcategories': [
+        {
+          'subcategory': subc,
+          'values': cats[i]['values']
+        }
+      ]
+    })
+  }
+}
+
+// console.log(newCategories)
+
+const newFileOutput = {
+  'Input': 'summary',
+  'key': ['size, type, units, min, max', "category;subcategory"],
+  'Output': newCategories,
+  'defaultHistoryLength': OldGraphData.defaultHistoryLength ?? 60
+}
```

Comment:
```suggestion
const newFileOutput = {
  'Output': newCategories,
  'defaultHistoryLength': OldGraphData.defaultHistoryLength ?? 60
}
```
It doesn't look like `Input` and `key` are used, so they can be removed, and `graph-data.json` can be regenerated.

Reply:
fixed